ext/redhat/facter.spec.erb

```diff
@@ -94,11 +94,11 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%{_bindir}/cfacter
+%{_bindir}/facter
 %{_libdir}/libfacter.so*
 %{_includedir}/facter
 %if 0%{?rhel} >= 7 || 0%{?fedora}
-%{vendor_ruby}/cfacter.rb
+%{vendor_ruby}/facter.rb
```

Comment:
I haven't really delved into that yet. Puppet will need to support either Facter 2 or Facter 3 though.